tests/componenttests/server/tests/mediaPipeline/VolumeTest.cpp

```diff
@@ -51,12 +51,24 @@ class VolumeTest : public MediaPipelineTest
                     *elementPtr = m_audioSink;
                 }));
 
-        EXPECT_CALL(*m_glibWrapperMock, gObjectClassFindProperty(_, StrEq("audio-fade"))).WillOnce(Return(nullptr));
-
-        EXPECT_CALL(*m_rdkGstreamerUtilsWrapperMock, isSocAudioFadeSupported()).WillOnce(Return(false));
-
+        // Normal volume change when volumeDuration == 0
+        if (m_volumeDuration == 0)
+        {
+            EXPECT_CALL(*m_rdkGstreamerUtilsWrapperMock, isSocAudioFadeSupported()).Times(0);
```

Comment:
Remove expect calls that require .Times(0), these are strict mocks so it is enforced automatically.